intellij-ext/src/main/kotlin/com/metalbear/mirrord/SemverCheck.kt

```diff
@@ -0,0 +1,63 @@
+package com.metalbear.mirrord
```

Comment:
Please rename file to VersionCheck.kt (semver is an implementation detail - also it should have been capitalized SemVer)

Reply:
thanks, done